src/Umbraco.Web.UI.Client/src/packages/core/modal/context/modal.context.ts

```diff
@@ -59,6 +63,8 @@ export class UmbModalContext<
 	#size = new UmbStringState<UUIModalSidebarSize>('small');
 	public readonly size = this.#size.asObservable();
 
+	public readonly interactionMemory = new UmbInteractionMemoryManager(this);
```

Comment:
Just reading a bit of code here :-)

This Modal Context is for all Modals?

Shouldn't it only be present for the Picker Modal?

It seems wrong to integrate this for all Modals, it may be a Custom Modal Context that is provided on the `picker-modal-base.element.ts`

A context using the same Context-Alias but a different API-Alias so if not present it will not look past the scope of the modal?

Reply:
I have removed the implementation inside the generic modal context. We know exchange memories through the "input-picker" and "picker-modal".